packages/tasks/src/library-ui-elements.ts

```diff
@@ -768,9 +768,9 @@ export const MODEL_LIBRARIES_UI_ELEMENTS: Partial<Record<ModelLibraryKey, Librar
 		snippets: stableBaselines3,
 	},
 	"ml-agents": {
-		btnLabel: "ml-agents",
+		btnLabel: "unity-ml-agents",
```

Comment:
```suggestion
		btnLabel: "ml-agents",
```

to be honest i don't remember how btnLabel is used, i think it's just analytics, so i would keep the current one here too